src/plugins/dashboard/public/dashboard_actions/filters_notification_popover.tsx

```diff
@@ -42,6 +42,7 @@ export function FiltersNotificationPopover({ api }: { api: FiltersNotificationAc
 
   const filters = useMemo(() => api.filters$?.value, [api]);
   const displayName = dashboardFilterNotificationActionStrings.getDisplayName();
+  const isSearchType = api.type === 'search';
```

Comment:
So we probably don't want to hardcode in an embeddable type like this, since it's not very flexible :) I think a better path forward would be to do this through the embeddable API. We have two paths I can see:

1. Adding an optional `canEditUnifiedSearch` method to the `PublishesUnifiedSearch` interface
2. Adding a new interface (something like `CanEditFilters` or `EditableUnifiedSearch`) that the saved search embeddable can implement to return `false`

Both solutions would have similar code on the `FiltersNotificationAction` side - it's more a question of where we want this code to live. I think I am leaning towards option 1 but I'm interested in what @nreese and/or @ThomThomson thinks 🤔

Reply:
> 1) Adding an optional canEditUnifiedSearch method to the PublishesUnifiedSearch interface that the saved search embeddable can implement to return false (the default would be true if this method isn't defined)

Option one is my preferred suggestion. Just make `canEditUnifiedSearch` optional so all embeddables do not need to implement the method.